src/PointwiseFunctions/Hydro/EquationsOfState/EquationOfState.hpp

```diff
@@ -32,7 +33,9 @@ namespace EquationsOfState {
 
 namespace detail {
 template <bool IsRelativistic, size_t ThermodynamicDim>
-struct DerivedClasses;
+struct DerivedClasses {
+  using type = tmpl::list<Spectral>;
```

Comment:
This should be in the `<true, 1>` case.  Add a test that calls `TestHelpers::test_creation<std::unique_ptr<EquationOfState<true, 1>>>("Spectral:\nargs...")`.